shared/product-make.include

```diff
@@ -88,7 +88,7 @@ $(OUT)/bash-remediations.xml: $(SHARED)/$(TRANS)/combineremediations.py $(bash_r
 	$(SHARED)/$(TRANS)/combineremediations.py $(PROD) $(BUILD_REMEDIATIONS) $(OUT)/bash-remediations.xml
 
 # Common build targets - an XCCDF with remediations but not linked to OVAL yet
-$(OUT)/xccdf-unlinked-withremediations.xml: $(OUT)/xccdf-unlinked-resolved.xml $(OUT)/xccdf-unlinked-ocilrefs.xml $(OUT)/bash-remediations.xml $(SHARED)/$(TRANS)/xccdf-addremediations.xslt
+$(OUT)/xccdf-unlinked-withremediations.xml: $(OUT)/xccdf-unlinked-ocilrefs.xml $(OUT)/xccdf-unlinked-resolved.xml $(OUT)/bash-remediations.xml $(SHARED)/$(TRANS)/xccdf-addremediations.xslt
```

Comment:
Because ```$(OUT)/xccdf-unlinked-resolved.xml``` doesn't contain ocil checks resolved to official OCIL-2.0 ones (there's still this ```ocil-transitional``` check system). While ```$(OUT)/xccdf-unlinked-ocilrefs.xml``` contains them already resolved (there's this ```$ocil_cs``` URL used as checksystem).

The point of this fix being NIST SCAP content validation test suite doesn't allow different check systems other than OVAL and OCIL. Since ```ocil-transitional``` is just SSG construct, we change the order to include remediations into that XCCDF, which is using official OCIL-2.0 check system (to silent the NIST suite).

Of course, we could simply remove the mentions about ```ocil-transitional```, but why to do that, when people might be accustomed to use it somehow. And at some point in the future, we in any case might want to support / produce valid OCIL.

So that's the reason (see also https://github.com/OpenSCAP/scap-security-guide/pull/1039#issuecomment-185254662 for the history how this evolved. [Original plan was to replace ```ocil-transitional``` with official OCIL-2.0. But this has shown not to be necessary, since SSG already produces XCCDF with valid OCIL -- so couple of patches from previous version have shown to be worthless]).